### PR TITLE
Pv2 2010 handle new period end messages

### PR DESCRIPTION
**Pull request by carlbm**

### CODE REVIEW

File: src/SFA.DAS.Payments.PeriodEnd.Application/Handlers/PeriodEndJobContextMessageHandler.cs [hunk 1]
```diff
@@ -64,7 +67,10 @@ public async Task<bool> HandleAsync(JobContextMessage message, CancellationToken
                 // PeriodEndStoppedEvent will be handled by the PeriodEndStoppedEventHandler which in turn is handled by the ProcessProviderMonthEndCommandHandler but we don't want to wait for it
 
 
-                if (periodEndEvent is PeriodEndStartedEvent || periodEndEvent is PeriodEndStoppedEvent)
+                if (periodEndEvent is PeriodEndStartedEvent 
+                    || periodEndEvent is PeriodEndStoppedEvent  
+                    || periodEndEvent is PeriodEndRequestReportsEvent  
+                    || periodEndEvent is PeriodEndRequestValidateSubmissionWindowEvent )
                 {
```
Comment (hardik-desai-kcl) [inline src/SFA.DAS.Payments.PeriodEnd.Application/Handlers/PeriodEndJobContextMessageHandler.cs:74]: please update this log message

File: src/SFA.DAS.Payments.PeriodEnd.AcceptanceTests/packages.config [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <package id="Autofac" version="4.8.1" targetFramework="net472" />
   <package id="ESFA.DC.CollectionsManagement.Models" version="0.2.9-beta0001" targetFramework="net472" />
   <package id="ESFA.DC.ILR.FundingService.FM36.FundingOutput.Model" version="0.1.446" targetFramework="net472" />
+  <package id="ESFA.DC.Jobs.Model" version="0.2.76-beta0001" targetFramework="net472" />
```
Comment (hardik-desai-kcl) [inline src/SFA.DAS.Payments.PeriodEnd.AcceptanceTests/packages.config:6]: why is this reference required

Comment (keenesolutions) [reply src/SFA.DAS.Payments.PeriodEnd.AcceptanceTests/packages.config:6]: This was required to fix the brokent component-level acceptance tests we are using to test event behaviour.

File: src/SFA.DAS.Payments.PeriodEnd.Application/Handlers/PeriodEndJobContextMessageHandler.cs (hunk 1 above)
Comment (keenesolutions) [reply src/SFA.DAS.Payments.PeriodEnd.Application/Handlers/PeriodEndJobContextMessageHandler.cs:74]: I've updated this.

**Pull request merged**